trikot-viewmodels-declarative-flow/sample/common/TRIKOT_FRAMEWORK_NAME.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'TRIKOT_FRAMEWORK_NAME'
-    spec.version                  = '3.3.7'
+    spec.version                  = '3.3.8'
```

Comment:
[Like @npresseault said](https://github.com/mirego/trikot/pull/74#discussion_r900072898), this shouldn’t be changed in pull requests.